pymc/tests/distributions/test_simulator.py

```diff
@@ -78,7 +78,7 @@ def test_one_gaussian(self):
         assert self.count_rvs(self.SMABC_test.logp()) == 1
 
         with self.SMABC_test:
-            trace = pm.sample_smc(draws=1000, chains=1, return_inferencedata=False)
+            trace = pm.sampling(draws=1000, chains=1, return_inferencedata=False)
```

Comment:
This is also the function, and shouldn't be changed (in all this file).

Reply:
Sorry for not being clear but this comment applies to the whole file. You should change back all of the places where you replaced `sample_smc` by `sampling` in this file.